<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QResultStatistics</class>
 <widget class="QWidget" name="QResultStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QResultStatistics</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1912</width>
     <height>793</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>1902</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1912</width>
      <height>60</height>
     </size>
    </property>
    <property name="bks" stdset="0">
     <string notr="true">gray</string>
    </property>
    <property name="borders" stdset="0">
     <string notr="true">all</string>
    </property>
    <widget class="QWidget" name="widget_1" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1841</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>70</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>日期</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QDateEdit" name="lower_date_edit">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>40</height>
               </size>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::YearSection</enum>
              </property>
              <property name="displayFormat">
               <string notr="true">yyyy/MM/dd</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>-</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="upper_date_edit">
              <property name="minimumSize">
               <size>
                <width>170</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
              <property name="currentSection">
               <enum>QDateTimeEdit::YearSection</enum>
              </property>
              <property name="displayFormat">
               <string notr="true">yyyy/MM/dd</string>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>模块</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_dev">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>项目名称</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_item_name">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>试剂批号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_reag_batch">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="query_btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>筛选</string>
          </property>
          <property name="buttons" stdset="0">
           <string notr="true">flat_btn</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_btn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>重置</string>
          </property>
          <property name="buttons" stdset="0">
           <string notr="true">flat_btn</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="graphic" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>128</y>
      <width>1422</width>
      <height>620</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_cards" native="true">
    <property name="geometry">
     <rect>
      <x>1509</x>
      <y>276</y>
      <width>398</width>
      <height>510</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>71</y>
      <width>1451</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <property name="spacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>19</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>样本量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resultRangeBtn">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>结果分段设置</string>
       </property>
       <property name="buttons" stdset="0">
        <string notr="true">flat_btn_blue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_recheck_enable">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>开启复查统计</string>
       </property>
       <property name="checks" stdset="0">
        <string>push</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="icon_recheck">
         <property name="text">
          <string>icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_recheck">
         <property name="text">
          <string>复查</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="icon_normal">
         <property name="text">
          <string>icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_normal">
         <property name="text">
          <string>常规</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_total" native="true">
    <property name="geometry">
     <rect>
      <x>1518</x>
      <y>80</y>
      <width>378</width>
      <height>195</height>
     </rect>
    </property>
    <property name="bks" stdset="0">
     <string notr="true">light_gray</string>
    </property>
    <property name="radiuses" stdset="0">
     <string>rds_4</string>
    </property>
    <widget class="QWidget" name="widget1" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>16</y>
       <width>291</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>25</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="text">
         <string>样本总量</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>00000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="fonts" stdset="0">
         <string notr="true">60</string>
        </property>
        <property name="weights" stdset="0">
         <string notr="true">bold</string>
        </property>
        <property name="colors" stdset="0">
         <string notr="true">light_blue</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>737</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>结果范围</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bk_graphic_label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>69</y>
      <width>1498</width>
      <height>719</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="borders" stdset="0">
     <string>all</string>
    </property>
   </widget>
   <widget class="QLabel" name="bk_sample_label">
    <property name="geometry">
     <rect>
      <x>1507</x>
      <y>69</y>
      <width>400</width>
      <height>719</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="borders" stdset="0">
     <string>all</string>
    </property>
   </widget>
   <zorder>bk_sample_label</zorder>
   <zorder>bk_graphic_label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>graphic</zorder>
   <zorder>widget_3</zorder>
   <zorder>widget_cards</zorder>
   <zorder>widget_total</zorder>
   <zorder>label_4</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
